routes/api/contacts.js

```diff
@@ -1,25 +1,20 @@
-const express = require('express')
+const express = require('express');
+const Joi = require('joi');
+const ctrl = require('../../controllers/contacts');
+const { validateBody } = require('../../middlewares');
 
 const router = express.Router()
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
```

Comment:
You should validate phone and email properly. Schema must be in separate file

Reply:
Corrected the comment